doc/release/README.chplenv.rst

```diff
@@ -111,6 +111,8 @@ MANPATH
 Optional Settings
 -----------------
 
+.. _chpl_target_platform:
```

Comment:
If all of the links to this section are within this file, I think you can just write "`CHPL_TARGET_PLATFORM`_" instead.